### PR TITLE
Added learning rate to lightGBM and update readme for mlflow ui tracking

### PR DESCRIPTION
**Pull request by shubham769**
## What changes are proposed in this pull request?
Similar to https://github.com/mlflow/mlflow/pull/2766, I was working on LightGBM and saw that the LightGBM example part should also change the same as xgboost.
I have added the learning rate in LightGBM example as the parameter play a huge role in optimization, moreover updated the readme by adding a way to do experiments and track them on mlflow UI

(Please fill in changes proposed in this fix)

## How is this patch tested?
I have tested this on my local machine and it is running fine
Also, I have tested the example test manually on my local
(Details)

### Is this a user-facing change?

- [ ] No. You can skip the rest of this section.
- [x] Yes. Give a description of this change to be included in the release notes for MLflow users.

This PR will make changes in the LightGBM example, which consists of the addition of an important parameter learning rate which plays a huge role in optimization. Moreover, Readme for the example is changed for the same modification and adding a way to do experiments and performance tracking with mlflow.

### What component(s), interfaces, languages, and integrations does this PR affect?
Components 
- [ ] `area/artifacts`: Artifact stores and artifact logging
- [ ] `area/build`: Build and test infrastructure for MLflow
- [x] `area/docs`: MLflow documentation pages
- [x] `area/examples`: Example code
- [ ] `area/model-registry`: Model Registry service, APIs, and the fluent client calls for
Model Registry
- [ ] `area/models`: MLmodel format, model serialization/deserialization, flavors
- [ ] `area/projects`: MLproject format, project running backends
- [ ] `area/scoring`: Local serving, model deployment tools, spark UDFs
- [ ] `area/tracking`: Tracking Service, tracking client APIs, autologging

Interface 
- [ ] `area/uiux`: Front-end, user experience, JavaScript, plotting
- [ ] `area/docker`: Docker use across MLflow's components, such as MLflow Projects and MLflow Models
- [ ] `area/sqlalchemy`: Use of SQLAlchemy in the Tracking Service or Model Registry
- [ ] `area/windows`: Windows support

Language 
- [ ] `language/r`: R APIs and clients
- [ ] `language/java`: Java APIs and clients

Integrations
- [ ] `integrations/azure`: Azure and Azure ML integrations
- [ ] `integrations/sagemaker`: SageMaker integrations

### How should the PR be classified in the release notes? Choose one:

- [ ] `rn/breaking-change` - The PR will be mentioned in the "Breaking Changes" section
- [x] `rn/none` - No description will be included. The PR will be mentioned only by the PR number in the "Small Bugfixes and Documentation Updates" section
- [ ] `rn/feature` - A new user-facing feature worth mentioning in the release notes
- [ ] `rn/bug-fix` - A user-facing bug fix worth mentioning in the release notes
- [x] `rn/documentation` - A user-facing documentation change worth mentioning in the release notes

### CODE REVIEW

**Comment by shubham769** [comment]
@sueann Can you please review this PR?  as you had already reviewed similar one

**Review by aarondav — APPROVED** [review]
LGTM - thanks!
